docs/cloud-edge/modules/_http_header_templates.mdx

```diff
@@ -126,21 +117,39 @@ Note: These values are only available when using the OAuth module.
 | `${.oauth.user.id}`       | The authenticated user's ID returned by the OAuth provider.                                   |
 | `${.oauth.user.name}`     | The authenticated user's name returned by the OAuth provider.                                 |
 
-### TLS Template Variables {#tls}
+#### OpenID Connect Variables {#openid-connect}
 
-Note: These values are only available when using the Mutual TLS module.
+These variables are only populated when using the OpenID Connect module. These
+variables are identical to the [OAuth Variables](#oauth).
 
-|                                     |                                                                                           |
-| ----------------------------------- | ----------------------------------------------------------------------------------------- |
-| `${.tls.cipher_suite}`              | The cipher suite selected during the TLS handshake.                                       |
-| `${.tls.client_cert.serial_number}` | The serial number of the client's leaf TLS certificate in the Mutual TLS handshake.       |
-| `${.tls.client_cert.subject.cn}`    | The subject common name of the client's leaf TLS certificate in the Mutual TLS handshake. |
-| `${.tls.client_cert.subject}`       | The subject of the client's leaf TLS certificate in the Mutual TLS handshake              |
-| `${.tls.version}`                   | The version of the TLS protocol used between the client and the ngrok edge.               |
+|                           |                                                                                                        |
+| ------------------------- | ------------------------------------------------------------------------------------------------------ |
+| `${.oauth.app_client_id}` | The is the ID of the OAuth application used to handle this request.                                    |
+| `${.oauth.decision}`      | 'allow' if the OpenID Connect module permitted the request to the upstream service, 'block' otherwise. |
```

Comment:
```suggestion
| `${.oauth.decision}`      | `allow` if the OpenID Connect module permitted the request to the upstream service, `block` otherwise. |
```